examples/existing_vpc/main.tf

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.0"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+#############################################################################
+# Provision VPC
```

Comment:
It probably doesn't make sense to provision the VPC and subnets in the example here, as this is not something someone using existing vpc + subnets resources would be doing. We could maybe have terraform code in a different folder that provisions the VPC + subnets, and have our test call that code first, and pass the VPC + subnet details to this example. We did this in the [TestRunExistingGroupExample](https://github.com/terraform-ibm-modules/terraform-ibm-resource-group/blob/0058f23ed3bab77970a20d5791d4bb705fc28a95/tests/pr_test.go#L41C6-L41C33) test if you want to use it for reference. 
Suggest getting the rest of the code up to date an working first before updating this though.